From d2972b15351605b8032a9b47f824a86a921093af Mon Sep 17 00:00:00 2001
From: "Sar Ashki, Babak" <Babak.SarAshki@windriver.com>
Date: Thu, 7 Nov 2019 11:38:18 -0800
Subject: [PATCH] rabbitmq-server build failure workaround

This package randomly fails to build when PARALLEL_MAKE is > 20.
It consistently builds with -j7. Actual solution is in the package
itself.
---
 recipes-extended/rabbitmq/rabbitmq-server_3.2.4.bbappend | 6 ++++++
 1 file changed, 6 insertions(+)
 create mode 100644 recipes-extended/rabbitmq/rabbitmq-server_3.2.4.bbappend

diff --git a/recipes-extended/rabbitmq/rabbitmq-server_3.2.4.bbappend b/recipes-extended/rabbitmq/rabbitmq-server_3.2.4.bbappend
new file mode 100644
index 0000000..92c7e05
--- /dev/null
+++ b/recipes-extended/rabbitmq/rabbitmq-server_3.2.4.bbappend
@@ -0,0 +1,6 @@
+# Fails to build and needs multiple attempts if PARALLEL_MAKE > 20
+# Work around for now: set PARALLEL_MAKE for this pacakge to 7
+# Actual fix is in the package itself
+
+EXTRA_OEMAKE_remove = " -j ${PARALLEL_MAKE}"
+EXTRA_OEMAKE = " -j7"
-- 
2.23.0

